addons/web/static/src/core/l10n/localization_service.js

```diff
@@ -67,6 +67,9 @@ export const localizationService = {
             }
         }
 
+        if (!userLocalization) {
+            return;
+        }
```

Comment:
This is probably not desirable, we should find a way to receive the missing information instead.